src/ecmult_impl.h

```diff
@@ -960,7 +960,7 @@ static size_t secp256k1_pippenger_scratch_size(size_t n_points, int bucket_windo
     size_t entries = n_points + 1;
 #endif
     size_t entry_size = sizeof(secp256k1_ge) + sizeof(secp256k1_scalar) + sizeof(struct secp256k1_pippenger_point_state) + (WNAF_SIZE(bucket_window+1)+1)*sizeof(int);
-    return ((1<<bucket_window) * sizeof(secp256k1_gej) + sizeof(struct secp256k1_pippenger_state) + entries * entry_size);
+    return sizeof(secp256k1_gej) << bucket_window + sizeof(struct secp256k1_pippenger_state) + entries * entry_size;
```

Comment:
```suggestion
    return (sizeof(secp256k1_gej) << bucket_window) + sizeof(struct secp256k1_pippenger_state) + entries * entry_size;
```

Reply:
I don't think excessive parentheses help readability.

Reply:
Except in this case they're not actually excessive; they're necessary for correctness. Sorry!